arm/Microsoft.Automation/automationAccounts/schedules/deploy.bicep

```
@@ -57,13 +57,13 @@ resource schedule 'Microsoft.Automation/automationAccounts/schedules@2020-01-13-
   name: name
   parent: automationAccount
   properties: {
-    advancedSchedule: (empty(advancedSchedule) ? null : advancedSchedule)
-    description: (empty(scheduleDescription) ? null : scheduleDescription)
-    expiryTime: (empty(expiryTime) ? null : expiryTime)
-    frequency: (empty(frequency) ? 'OneTime' : frequency)
-    interval: ((0 == interval) ? null : interval)
-    startTime: (empty(startTime) ? dateTimeAdd(baseTime, 'PT10M') : startTime)
-    timeZone: (empty(timeZone) ? null : timeZone)
+    advancedSchedule: !empty(advancedSchedule) ? advancedSchedule : null
+    description: !empty(scheduleDescription) ? scheduleDescription : null
+    expiryTime: !empty(expiryTime) ? expiryTime : null
+    frequency: !empty(frequency) ? frequency : 'OneTime'
+    interval: (0 != interval) ? interval : null
```

Reads better, and its lot having the same 'got-ya' as PowerShell.
```suggestion
    interval: (interval != 0) ? interval : null
```

---

done